lib/mini_profiler/profiler.rb

```diff
@@ -331,8 +331,12 @@ def call(env)
       page_struct[:root].record_time((Process.clock_gettime(Process::CLOCK_MONOTONIC) - start) * 1000)
 
       if flamegraph
-        body.close if body.respond_to? :close
-        return client_settings.handle_cookie(self.flamegraph(flamegraph))
+        if query_string =~ /save/
+          ::File.write 'flamegraph.html', flamegraph
```

Comment:
I think you should write this to a file in `tmp` and then output where it was saved eg: 

```
saved flamegraph to tf23.docker  /tmp/flamegraphs_20190201-01023.html`
```

You get the hostname from `hostname`